### DISCUSSION TITLE
Add Stripe as a payment gateway

### DISCUSSION THREAD

**Discussion by jonassiewertsen** [jonassiewertsen/statamic-butik#163] [Ideas]
As @nn-dmt did point out, Mollie might not be the best option for USA vendors: https://github.com/jonassiewertsen/statamic-butik/issues/161

> I am forced to use either Stripe or Paypal because i'm in the USA.
> 
> I am not a php developer and so the documentation provided does not do me any good - it talks like a lot of assumptions made that I'm already very well versed in php/laravel developer - I am someone who is coming from html/css/wordpress and trying to make a working site that i can very easily make in wordpress/woocommerce but in an alternative way 'from scratch'
> 
> Is stripe / paypal payment gateways on your roadmap? 

This is the reason I am adding it as an idea.

**Integrating Stripe as a first-party integration is on the roadmap**

**Comment by ghost** [comment]
However I can help support getting stripe integration done faster let me know.

If I come up with an integration before you can I will inform you so you can check if you want to use what I build

**Reply by ghost** [reply]
I want to also suggest looking at WAVE as an integration as well. (https://waveapps.com

wave accounting handles charges, and book keeping. Having an integration with wave would eliminate the middle man of using stripe and then wave for many people who want to only use one solution.

**Reply by jonassiewertsen** [reply]
Yes, please let me know if you come up with a Stripe integration before myself.

**Comment by ghost** [comment]
I was reading their documentation, and it seems like they have a few different ways of integrating their API - 

However, for Butik - this method may be the easiest to implement:
https://stripe.com/docs/payments/accept-a-payment?integration=checkout
